source/Calamari.Tests/Fixtures/Nginx/Approved/NginxFixture.SetupReverseProxyWithSslUsingCertificateVariableSite.crt.approved.txt

```diff
@@ -0,0 +1,63 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
This cert is just a test certificate (that is expired) and doesn't expose anything secret